src/main/java/io/swagger/codegen/v3/generators/java/SpringCodegen.java

```diff
@@ -77,7 +77,7 @@
     protected boolean useOptional = false;
     protected boolean openFeign = false;
     protected boolean defaultInterfaces = true;
-    protected String springBootVersion = "1.5.22.RELEASE";
+    protected String springBootVersion = "2.1.16.RELEASE";
```

Comment:
2.1.x is still supported for a few months at least (https://spring.io/blog/2019/12/10/spring-boot-2-1-x-eol-november-1st-2020)